release/k3s/k3s.go

```diff
@@ -203,6 +203,7 @@ func (r *Release) SetupK8sRemotes(_ context.Context, ghClient *github.Client) er
 }
 
 func (r *Release) RebaseAndTag(_ context.Context, ghClient *github.Client) ([]string, string, error) {
+	disableGpgSigning := true
```

Comment:
I don't know how necessary this variable is here. There's no way to override it anywhere so it'd make more sense to not define it here or in the function below.

Reply:
I'd remove the variable assignment as it appears as though there is no way it would be any other value other than `true`. It adds more code than necessary as well as adds a code switch that doesn't need to be checked.